task/01-strings-tasks.js

```diff
@@ -69,7 +69,7 @@ function getStringFromTemplate(firstName, lastName) {
  *   'Hello, Chuck Norris!' => 'Chuck Norris'
  */
 function extractNameFromTemplate(value) {
-    throw new Error('Not implemented');
+    return value.slice(7, value.length - 1);
 }
```

Comment:
looks good. Please note you can also use negative numbers in slice. In the example ` value.slice(7, -1);` can do the same